### DISCUSSION TITLE
23.11.23 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#77] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 커스텀 애노테이션을 타지 않는 문제 해결 (with. 자현님🙇‍♀️)
    - 발표 준비
- 내일은 어떤 일을 하나요?
    - 발표 준비
    - 우재님 코드 리뷰

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 알림 기능 연결  부분 완료
  - PR 생성
- 내일은 어떤 일을 하나요?
  - 발표 준비
  - 코드 개선

**Reply by Jeonwoojae** [reply]
친구라면 공개 플레이리스트는 조회 가능하도록 변경.
